CHANGELOG.md

```diff
@@ -136,6 +136,19 @@ If you're passing custom HTML into the cookie banner component (for example, usi
 
 This change was introduced in [pull request #2432: Remove default font styles from cookie banner Sass](https://github.com/alphagov/govuk-frontend/pull/2432).
 
+#### Import 'base' before importing Sass files from `core` or `overrides` layers
+
+If you import individual files from the `core` or `overrides` layers, you might see the error `Undefined mixin 'govuk-exports'` or `no mixin named govuk-exports` when compiling your Sass.
```

Comment:
@vanitabarrett Doubt this is anything we need fix, but just wondering - why are there quotes around govuk-exports in the first mention, but none around the second? Is this just how it displays in the terminal?

Reply:
@EoinShaughnessy That's how the error displays in the terminal: `Error: no mixin named govuk-exports`

Reply:
@vanitabarrett Ah cool, cheers for clarifying!